data/ka'het/ka'het missions.txt

```diff
@@ -293,3 +293,393 @@ mission "Ringworld Debris: Quarg"
+			choice
+				`	"If they can duplicate themselves, why aren't there more Ka'sei here?"`
+					goto kasei
+				`	"If they were able to get here, does this suggest they have Jump Drives?"`
+				`	"Why aren’t they in other systems?"`
+			label jumpdrive
+			`	"They do not have any drive on board, not even a hyperdrive. Limited to sub-light velocity, they would need hundreds of years to pass from one system to another, which is a testament to the Builders' patience. Most species we have encountered abandon any form of sub-light transportation between star systems immediately after developing hyperdrive technology. For all we know, the Ember Waste could have been visited by these Fetri'sei or some other Ka'sei, but they had to leave for some reason we are not aware of. The last Builder died almost 100,000 years ago - so long that having a clear idea of what happened is impossible.`
+			choice
+				`	"What's the difference between Ka'sei and Fetri'sei?"`
+			`	"Well, there is not," Taely responds, "not in this case at least. The naming scheme the Builders used is a bit odd: we know that 'Ka', in their language, has a meaning close to the one of 'group', 'team' and sometimes even 'ring', although it can never be used without a name next to it. So we have 'group mar', 'group het', and so on. These are not limited to spaceships: the Ka'het, for example, includes both the biological spacecrafts around the Graveyard and the stations that constructs them, while until now we thought that the Ka'sei included only a handful of large scale construction projects. We know 'Fetri' to be a sort of spaceship sub-classification, but we are not aware of its exact meaning. So the Fetri'sei, or 'ship sei' are a part of the Ka'sei, or 'group sei.' To put it simply, it is similar to how all squares are rectangles, but not all rectangles are squares. It just so happens that everything in this system is classified as a Fetri'sei, so we can use either interchangeably.`
+			`	"I know a few people who will want to see all this. I will contact ahead and arrange to meet them on Aventine."`
+
+	on complete
+		"remnant met dusk" ++
+		conversation
+			`You complete the post-flight checks and disembark down the ship with Taely. "So, what do we do now?" you ask.`
+			`	"I fear I have to go back to my job now, Captain," she replies. "I would love to continue the exploration with you, but there are more pressing issues I have to resolve at the moment. I will keep an eye on the activity in Patir, I promise."`
+			`	As you speak to Taely, a young Remnant with a touchpad walks towards you and waves. Taely says, "Captain <first>, let me introduce you to Dusk. He is a young researcher who specializes in xenoarchaeology and astrobiology. Dusk, this is <first>, the captain who I told you about earlier."`
```

Comment:
what if you already met dusk? the next part of the remnant campaign pr has that happening I think so may as well consider that possibility

Reply:
I asked zitchas, he agrees to add those in Cognizance II since this one is likely to be merged earlier than that

Reply:
This has the basics of compatibility with Cognizance II, but I'm more than happy to make whatever compatibility changes are needed in Cognizance II, so this can merge right away without needing to try to synchronize anything with my PR.